pulser-core/pulser/sequence/_seq_drawer.py

```diff
@@ -718,61 +837,6 @@ def phase_str(phi: float) -> str:
         ]
         ax_lims = [ax_lims[i] for i in ch_data.curves_on_indices()]
 
-        if draw_phase_area:
-            top = False  # Variable to track position of box, top or center.
-            print_phase = not draw_phase_curve and any(
-                seq_.type.phase != 0
-                for seq_ in seq._schedule[ch]
-                if isinstance(seq_.type, Pulse)
-            )
-            for pulse_num, seq_ in enumerate(seq._schedule[ch]):
-                # Select only `Pulse` objects
-                if isinstance(seq_.type, Pulse):
-                    if sampling_rate:
-                        area_val = (
-                            np.sum(yseff[0][seq_.ti : seq_.tf]) * 1e-3 / np.pi
-                        )
-                    else:
-                        area_val = (
-                            np.sum(
-                                sampled_seq.channel_samples[ch].amp[
-                                    slot.ti : slot.tf
-                                ]
-                            )
-                            * 1e-3
-                            / np.pi
-                        )
-                    phase_val = seq_.type.phase
-                    x_plot = (seq_.ti + seq_.tf) / 2 / time_scale
-                    if (
-                        seq._schedule[ch][pulse_num - 1].type == "target"
-                        or not top
-                    ):
-                        y_plot = np.max(seq_.type.amplitude.samples) / 2
-                        top = True  # Next box at the top.
-                    elif top:
-                        y_plot = np.max(seq_.type.amplitude.samples)
-                        top = False  # Next box at the center.
-                    area_fmt = (
-                        r"A: $\pi$"
-                        if round(area_val, 2) == 1
-                        else rf"A: {area_val:.2g}$\pi$"
-                    )
-                    if not print_phase:
-                        txt = area_fmt
-                    else:
-                        phase_fmt = rf"$\phi$: {phase_str(phase_val)}"
-                        txt = "\n".join([phase_fmt, area_fmt])
-                    axes[0].text(
-                        x_plot,
-                        y_plot,
-                        txt,
-                        fontsize=10,
-                        ha="center",
-                        va="center",
-                        bbox=area_ph_box,
-                    )
-
         # Draw target regions phase_shifts
         if draw_phase_shifts:
```

Comment:
In the end, why don't you pass `_basis_ref` to `SequenceSamples` when you sample a `Sequence` (just like `_measurement` and `_magnetic_field`) ?

Reply:
Yes, I can do this.

Reply:
I did; if I didn't miss anything should be ok now; it is still missing coverage, but I wait for your approval first

Reply:
I am noticing an issue in the tutorial [Output Modulation and EOM](https://pulser.readthedocs.io/en/stable/tutorials/output_mod_eom.html#). The buffers before and after the EOM mode is switched on (the time slots that are just before and just after the eom interval) are not represented...
Obtained:
![image](https://github.com/pasqal-io/Pulser/assets/61453516/c62df160-47f7-4d13-918d-198a90baf276)
Should be:
![image](https://github.com/pasqal-io/Pulser/assets/61453516/a572e8c7-c734-4d59-9ca2-5fbfd0c85e98)

Reply:
I see; I think the problem is in the `gather_data` part where we create eom buffers starting from `ChannelSamples.target_time_slots`; the drawing part is almost unchanged

Reply:
I further investigate it, but I cannot figure out what is the problem.